sourcecode/hub/lang/en/messages.php

```diff
@@ -60,4 +60,8 @@
     'version-history' => 'Version history',
     'published' => 'Published',
     'change-email' => 'Change email',
+    'disconnect-social-accounts' => 'Disconnect social accounts',
+    'disconnect-google' => 'Google',
```

Comment:
The buttons labels should describe the whole action, for instance 'Disconnect from Google'.